### ISSUE TITLE
What checks does the TRC process around selection of members from IIIF-C associate members and the community?

### ISSUE THREAD

**Issue by zimeon** [IIIF/website#124]
The current proposal has TRC Ex Officio members and Consortium members representing IIIF-C Full members making the selection members from IIIF-C associate members and the community _"with a view to maximising the diversity of representation amongst those who have previously engaged with the specification process."_ Do we need any addition requirement to avoid favoritism or popularity bias?

**Comment by azaroth42** [comment]
I do not believe that it does. The members should be trusted to act in good faith, and the size of the consortium full membership is high enough to ensure that loud voices do not unduly influence the results.

**Comment by zimeon** [comment]
Discussion on [2018-08-29 Technical Community Call](https://docs.google.com/document/d/1Rx-sUh2STNf6_GxRC1ghqsEuoj5LymRDsxi0_WpqCJ4/edit) - no voices in favor of an additional requirement
